python/pylibcudf/pylibcudf/libcudf/wrappers/timestamps.pxd

```diff
@@ -1,9 +1,12 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
-from libc.stdint cimport int64_t
+from libc.stdint cimport int32_t, int64_t
```

Comment:
This would look like after you `cimport` from `durations.pxd`
```suggestion
from libc.stdint cimport int32_t, int64_t


cdef extern from "cudf/wrappers/timestamps.hpp" namespace "cudf::detail" nogil:
    cdef cppclass time_point[duration]:
        pass
    cdef cppclass timestamp[duration]:
        pass
```